a00_qpip/plugin.py

```diff
@@ -251,9 +251,13 @@ def pip_install_reqs(self, reqs_to_install):
         os.makedirs(self.prefix_path, exist_ok=True)
         log(f"Will pip install {reqs_to_install}")
 
+        # python is normally found at sys.executable, but there is a bug on windows qgis so use 'python' instead
+        # https://github.com/qgis/QGIS/issues/45646
+        python_command = 'python' if os.name == 'nt' else sys.executable
```

Comment:
Just to be sure, is this tested?
I guess the python command is on `PATH`, so it looks plausible.

Reply:
I tested it on my machine and it worked on Windows. However, it would be good if it could be tested by others as it could depend on how the machine is setup (I have another installation of python on my machine, for example).

Reply:
You could print `sys.executable` from within the python subprocess to validate which one is started?

Reply:
```suggestion
        python_command = "python" if os.name == "nt" else sys.executable
```

let's make our linter happy too